### ISSUE TITLE
Cannot upgrade walls, only level 1 (medievalOak)

### ISSUE THREAD

**Issue by boriandrea1994** [ldtteam/minecolonies#4809]
### Minecolonies version

Version: 
- Minecolonies: minecolonies-0.10.771-RELEASE-universal.jar
- Minecraft: 1.15.2
- Forge: 31.1.0

### Expected behavior
  

- I take the builder to level 3.
- I request the construction of a level 3 schematic gate.
- The builder builds the wall and gives me the opportunity to upgrade to 4 or 5 as soon as the builder is at these levels.
- The same for all other "decorations".

### Actual behaviour

- I would like to build the level 3 gate.
- I use the tool to commission the construction of the gate at level 3.
- In the logs, the message reports the construction of level 1 and not of level 3, as required.
- After finishing the construction of the gate at level 1, if I click the "panel" at the bottom to upgrade, I can't do anything and anything I click doesn't work.
- If I try to ask the builder (using the tool) to build a level 3 gate above level 1, everything stops and the builder no longer works. In his building, only the level 1 gate is 100% completed.
- If you look at the screenshots you can see that the builder only takes level 1.

### Steps to reproduce the problem

1. Use the creative mode (to give all the resources!)
2. To repeat the report, please load the save (attached to the report) and try to build the gate somewhere.
3. After the builder finishes, try to upgrade the gate.
4. The gate cannot be upgraded anymore.

### Logs
- latest.log (https://gist.github.com/boriandrea1994/0002c6ffdcd126007bf33d18793f6d71)
- world & screens (https://filebin.net/9lrbxr01expicex4)

Sorry for the attachment, i know they can stay there for 1 week. If you have another way to share files, please, tell me :)

Thank you very much!

**Comment by Raycoms** [comment]
Can you test this in the latest alpha?

**Comment by boriandrea1994** [comment]
> Can you test this in the latest alpha?

U.. I don't know where to start to configure the game with the alpha version.
I don't want to mess with the current version.

is it possible that the problem is limited to the type of building (oak?)

I see a lot of exceptions in the logs regarding:
at com.mojang.datafixers.DataFixerUpper.update (DataFixerUpper.java:86) [datafixerupper-2.0.24.jar :?]

Is there anything else I can do to help you with the checks?

**Comment by Raycoms** [comment]
there should be no problems arising from downloading the latest minecolonies/structurize versions and putting them in your mod folder.

**Comment by boriandrea1994** [comment]
> there should be no problems arising from downloading the latest minecolonies/structurize versions and putting them in your mod folder.

Ok then I proceed.

After downloading the latest version of minecolonies, can I reload my current world or do I have to recreate a new one?

Where i can get the new version for minecolonies/structurize?

**Comment by Raycoms** [comment]
You can just continue playing on that alpha of minecolonies. It is very stable so you shouldn't run into any problems. You can download it on curseforge.

(If you want to roll back to the previous version again you should do a world backup before adding the new minecolonies version)

**Comment by boriandrea1994** [comment]
Ok then, I installed the alpha version.

Even indicating the construction of a higher level, the builder imposes himself starting from level 1.
However, in the alpha version, I now have the option to upgrade.
I have to build walls one level at a time.

I think the report at this point can be closed, if I did not understand what the problem was.

Just one last thing; changing the version works, however is it correct that I have to build step by step all the levels?
